README.md

```diff
@@ -735,7 +746,7 @@ A curated list of awesome C/C++ frameworks, libraries, resources, and shiny thin
 
 * [Cppcheck](http://cppcheck.sourceforge.net/) - A tool for static C/C++ code analysis. - [source](https://github.com/danmar/cppcheck)
 * [PVS-Studio](http://www.viva64.com/en/pvs-studio/) - A tool for bug detection in the source code of programs, written in C, C++ and C#.
-* [include-what-you-use](https://code.google.com/p/include-what-you-use/) - A tool for use with clang to analyze includes in C and C++ source files.
+* [include-what-you-use](https://github.com/include-what-you-use/include-what-you-use) - A tool for use with clang to analyze includes in C and C++ source files. [website](https://include-what-you-use.org/)
```

Comment:
https://github.com/include-what-you-use/include-what-you-use/blob/master/LICENSE.TXT

Reply:
I also noticed that, but I am not sure whether it is the LICENSE for `include-what-you-use` or just because `include-what-you-use` uses `clang` so the `LICENSE.txt` is for that. So I just ignored that.